src/App.js

```diff
@@ -2,14 +2,14 @@ import React, { Component } from 'react';
 import { BrowserRouter, Route, Switch } from 'react-router-dom';
 
 import DetailPage from './pages/DetailPage';
-import SearchPage from './pages/SearchPage';
+import SearchPage from './pages/SearchPage/index';
```

Comment:
Se o arquivo dentro da pasta se chama `index.js` você só precisa apontar a pasta. É isso mesmo o que você quer?
```suggestion
import SearchPage from './pages/SearchPage';
```